js/data/item_data.js

```diff
@@ -354,3 +360,11 @@ function getNaturalGift(item) {
     }
     return {'t':'Normal','p':1};
 }
+function getTechnoBlast(item) {
```

Comment:
Seems like this should be done the same way as `getItemBoostType(item)`, just returning the appropriate type.